rpc/nns/names.go

```diff
@@ -0,0 +1,13 @@
+package nns
+
+// A set of standard contract names deployed into NeoFS sidechain.
+const (
```

Comment:
We have a set of names in neofs-node repo (https://github.com/nspcc-dev/neofs-node/blob/920743715ddfcfd882011f5924c5476bf9e940bd/cmd/neofs-adm/internal/modules/morph/initialize_deploy.go#L39-L49). Maybe it's a good idea to remove them from neofs-node and use these constants instead?

Reply:
Where do you think I've copy-pasted them from?! Of course it should be done during upgrade to 0.18.0.